package.json

```diff
@@ -15,7 +15,7 @@
     "client:deploy:nc": "npm run client:deploy -- --no-confirm",
     "client:build:deploy": "npm run build && npm run client:deploy",
     "client:build:deploy:nc": "npm run build && npm run client:deploy:nc",
-    "cloudfront:setup": "sls deploy",
+    "setup": "sls deploy",
```

Comment:
This change will affect other scripts. Please either keep the old command or update other scripts accordingly.

Reply:
> This change will affect other scripts. Please either keep the old command or update other scripts accordingly.

Fixed. Thanks